### DISCUSSION TITLE
What's the proper process for updating config.yaml and restarting the manager with the new settings?

### DISCUSSION THREAD

**Discussion by raid5** [swar/Swar-Chia-Plot-Manager#84] [Q&A]
More specifically I'm curious how to do this while the manager has already kicked off and is monitoring existing plotting processes. Thanks!

**Comment by noodlesong** [comment]
You can do `python manager.py restart`, the plots already in progress will keep working.

**Reply by raid5** [reply]
I heard this has issues if you need to change directories, but I was wondering if someone could confirm. As an example, I updated my destination to a newly purchased usb drive, then ran the `restart` command, but the view is showing `?` for the job name for the plots that were already in progress. Is this just a visual issue/bug, or more serious?

Is there a better process for when you need to change directories in the config?

<img width="789" alt="Screen Shot 2021-05-04 at 2 19 03 AM" src="https://user-images.githubusercontent.com/15592/117171263-92c88a00-ad7f-11eb-89ff-19a96e31248c.png">

**Reply by noodlesong** [reply]
Seems to be a bug in the view, you can just restart it. Plots aren't affected.
